user-guides/data-farming/README.md

```diff
@@ -0,0 +1,49 @@
+---
+description: Learn the ins-and-outs Data Farming
+---
+
+# Data Farming Guides
+
+<figure><img src="../../.gitbook/assets/gif/farming.gif" alt=""><figcaption></figcaption></figure>
+
+Data Farming is the core incentive system inside of Ocean Protocol. It offers different ways for participants to help the protocol grow and succeed.
+
+It's broken down into 2 reward streams:
+1. Passive Rewards
+2. Active Rewards
+
+As the name suggests, users need to do very little to earn Passive Rewards and the program is aimed to onboard as many people as possible to help drive liquidity and conviction into the protocol.
+
+Active Rewards is an evolving incentive program that offers different venues for contributors to complete active work that drives the protocol forward.
+
+Anyone is welcomed to participate.
+
+## Passive Rewards stream
+
+The primary objective of the [Passive Rewards](../../rewards/df-intro.md#what-are-passive-rewards) stream is to provide an accessible way for anyone to use OCEAN and help the protocol grow.
+
+In [the veOCEAN user guide](user-guides/data-farming/how-to-farm-veOCEAN.md), you'll learn more about the Passive Rewards stream and how to get veOCEAN.
```

Comment:
this link is broken. I imagine it needs to have ../ in the begining

Reply:
done